lib/ui/natives.dart

```diff
@@ -45,6 +49,10 @@ void saveCompilationTrace(String filePath) {
   file.writeAsBytesSync(result);
 }
 
+/// Returns Dart compilation trace as a memory buffer.
+///
+/// The output is a list of symbols compiled by the Dart JIT at runtime up to this point,
+/// and can be passed later to Dart tools in order to precompile this code offline.
 List<int> dumpCompilationTrace() {
```

Comment:
the docs here should answer the following questions also:

* what modes (profile, release, debug...) does this do anything useful in
* what happens in the other modes
* what's the actual format of the data?
* what are the tools that you can pass this to?
* can this be depended on long-term?
...and maybe others, i don't know what the questions to ask are. :-)

Reply:
Done